### ISSUE TITLE
Model output seems not consistent.

### ISSUE THREAD

**Issue by ms-ko** [mlfoundations/open_clip#351]
Hi, 
Thank you for sharing the great work.
With the RN50 model , I found that the output features are not consistent between when I processed batch-wise and when I processed sample-wise (the total data samples I used here are the same for the both cases. I used 32 pairs of image and text and processed the 32 pairs together or separately). 

However, if I used ViT-L-14, I got the same output features. 
Did anyone encounter the same issue?

Moreover, I want to know how to load the same batch of data if I use the dataloader provided in this github. 
Based on my experiment, it seems that the loaded data samples are different whenever I run the script again, even with the same seed. Does anyone know how to address this?

Thanks!

**Comment by rwightman** [comment]
@ms-ko did you set the model to .eval()? ResNet models have batch norm in them which will vary with batch composition if you haven't set to eval mode (common PyTorch gotcha)
